docs/products/access/release-notes/1.x.md

```diff
@@ -1,5 +1,31 @@
 # Release notes 1.x versions
 
+## 1.1.1
+
+### Bug fixes
+
+* Resolved an issue causing some of the Access Admin's templates from being rendered correctly.
```

Comment:
Should we even mention fixing a bug that didn't make it to production?

Reply:
Good one, it doesn't add any value. Remove 👍

Reply:
Moved the contents to the index.md